src/main/scala/tech/cryptonomic/conseil/tezos/DatabaseConversions.scala

```diff
@@ -306,22 +313,11 @@ object DatabaseConversions {
         kind = kind,
         blockHash = block.data.hash.value,
         blockLevel = block.data.header.level,
-        timestamp = toSql(block.data.header.timestamp)
+        timestamp = toSql(block.data.header.timestamp),
+        internal = op.internal
       )
   }
 
-  implicit val blockToOperationsRow = new Conversion[List, Block, Tables.OperationsRow] {
```

Comment:
Wasn't this actually used? Very good catch, then!

Reply:
It wasn't used and at first I spent some time trying to figure out why it's not working 😄